src/kv/change_set.h

```diff
@@ -18,7 +18,7 @@ namespace kv
   using Snapshot = champ::Snapshot<K, VersionV<V>, H>;
 
   template <typename K>
-  using Read = std::map<K, Version>;
+  using Read = std::map<K, std::tuple<Version, Version>>;
```

Comment:
This really needs documenting. With one version, it's reasonably obvious, but not with two.

Reply:
Seconded, ideally as a named `struct`.